daml/Marketplace/Custody/Service.daml

```diff
@@ -130,11 +131,12 @@ template Service
           Custody.TransferDepositRequest{accountId; transfer} <- fetchAndArchive transferDepositRequestCid
           exerciseByKey @AssetSettlementRule accountId transfer
 
-      nonconsuming Lifecycle : [ContractId AssetDeposit]
+      nonconsuming Lifecycle : (ContractId AssetDeposit, [ContractId SettlementInstruction])
         with
           lifecycleRequestCid : ContractId Custody.LifecycleRequest
           safekeepingDepositCid: ContractId AssetDeposit
           fixings: Map Text (Map Date Decimal) --TODO these should be looked up from a contract on-ledger, not provided by the custodian!
+          uniquePayoutId: Text
```

Comment:
following the style from the rest of the marketplace model, type declarations should be `varName : Type`, not `varName: Type`.

Reply:
actually, there does seem to be some inconsistency across the model, however this file all has the space before the `:` so I would go with that here

Reply:
Should I modify 136-138 as well?

Reply:
yeah that would be great, thank you 👍